src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -695,7 +713,19 @@ public DescriptorImpl() {
             if (repoJobs == null) {
                 repoJobs = new ConcurrentHashMap<String, Set<AbstractProject<?, ?>>>();
             }
-            save();
+            saveAfterPause();
+        }
+        
+        private void saveAfterPause() {
```

Comment:
So many times while I am trying to develop locally Jenkins just wouldn't start.  It would hang before loading all plugins finishes.  I was debugging through, and noticed that if I caused enough delay after instantiating the Descriptor, just before the logger, then it would load.  Once I added this pause it never hangs anymore.  I think it was getting into an infinite loop or something depending on how long it takes to start up.